src/spikeinterface/core/baserecording.py

```diff
@@ -1,18 +1,22 @@
-from typing import Iterable, List, Union
-from pathlib import Path
 import warnings
+from pathlib import Path
+from typing import Iterable, List, Union
+from warnings import warn
```

Comment:
Thank you for the re ordering. This is a huge contribution.

Reply:
😆 sorry forgot to turn isort off